### PR TITLE
West Midlands - Class 2 - Michael Biruk - Git and GitHub - Week 1

### PR DESCRIPTION
**Pull request by mikinet**
# Your Details

- Your Name: Michael Biruk
- Your City: Gateshead
- Your Slack Name: Michael

# Homework Details

- Module: Git and GitHub
- Week: 1

### CODE REVIEW

**Comment by lish-dev** [comment]
Nice work Micheal! Would you look into adding another answer for q.18? :)

File: homework.md [hunk 1]
```diff
@@ -44,61 +44,63 @@ Using CSS, how would I select every `<h1>` tag on the page?
...
 
-Send you code to Github
+Save the changes you made to a file in GitHub
 
 ## Question 16
 
 What program do we use to write code at CodeYourFuture?
 
-Microsoft Word
+Microsoft Visual Studio Code
```
Comment (lish-dev) [inline homework.md:104]: Hi Michael, nice work! Would you look into adding another solution for question 18?

File: homework.md [hunk 2]
```diff
@@ -44,61 +44,63 @@ Using CSS, how would I select every `<h1>` tag on the page?
...
+1. It makes file sharing easy
+2. Tracking changes made to a file is simple
+3. It facilitates collaboration and team work
 
 ## Question 9
 
 What happens when you `fork` a repository?
 
-You delete it
+You copy it
```
Comment (ironkamosi) [inline homework.md:61]: RE: Question 9
Hi Biruk,

The answer for the question is correct,. 
Could you give more details about what happens in the process of a repository. 

Aaron

Comment (mikinet) [reply homework.md:61]: Hi Aaron,
I'll check it and try to amend it. Thanks for the feedback.

File: homework.md (hunk 1 above)
Comment (mikinet) [reply homework.md:104]: Hi Lisha,
Thanks for the feedback. I'll check that one.

**Pull request closed without merging**